README.rst

```diff
@@ -138,10 +138,14 @@ To run the Python syntax and format compliance tests::
 
   $ tox -e pep8
 
-To run the test suite against Python 2.7::
+To run the unit test suite against Python 2.7::
 
   $ tox -e py27
 
+To run the integration test suite against the default Python installation:
+
+  $ tox -e integration
+
```

Comment:
Actually, right now the integration tests automatically start up the PyKMIP server. We'll remove that once we put in the dynamic config for talking to (possibly) remote appliances.

I'll update the README with instructions on how to start the server once we make that change.